Resources/Private/Fusion/Prototypes/LabelRenderer.fusion

```diff
@@ -0,0 +1,23 @@
+prototype(Neos.Fusion.Form:LabelRenderer)  < prototype(Neos.Fusion:Component) {
+
+    @propTypes {
+        for = ${PropTypes.string}
+        label = ${PropTypes.string}
+        class = ${PropTypes.string}
+        translationPackage = ${PropTypes.string}
+        translationSource = ${PropTypes.string}
+    }
+
+    for = null
+    label = null
+    class = null
+
+    translationPackage = null
+    translationSource = null
+
+    renderer = afx`
+        <label for={props.for} class={props.class}>
+            {I18n.id(props.label).package(props.translationPackage).source(props.translationSource).value(props.label).translate()}
```

Comment:
Didn't we want to use fieldname here?

Reply:
Switched to label as there are cases where you want an container but nut the label. That way by passing label you trigger rendering and translation.